src/anm/media/audio.js

```diff
@@ -321,19 +321,21 @@ Audio.prototype.toggleMute = function() {
     }
 };
 /** @private @method connect */
-Audio.prototype.connect = function(element) {
+Audio.prototype.connect = function(player) {
     var me = this;
-    element.on(C.X_START, function() {
+    player.on(C.S_PLAY, function() {
```

Comment:
I thought on changing these events to be fired from `Animation` class. Like `ANIM_START`, `ANIM_END`, similar to those (`X_START`/`X_END`) in the element, this way it won't be required to pass player here.

Also, `X_START`/`X_END` shouldn't be removed here and in `video.js`, they're the events fired when the Element lifetime (timeband) starts or ends (not a player starts/ends), Audio & Video should both be stopped in these cases.